### PR TITLE
Remove link and remove cluster name

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
https://bugzilla.redhat.com/show_bug.cgi?id=2223702

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
/retitle Bug 2223702: Remove link and remove cluster name

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-08-01 15-03-51

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar, @GowthamShanmugam Please review

**Comment by SanjalKatiyar** [comment]
shorten commit title and fix build test...

File: packages/shared/src/dashboards/utilization-card/area-chart.tsx [hunk 1]
```diff
@@ -128,6 +128,15 @@ export const AreaChart: React.FC<AreaChartProps> = ({
     CustomChartGroup = ChartGroup;
   }
 
+  const removeParenthesesFromLegendData = (legData) => {
```
Comment (bipuladh) [inline packages/shared/src/dashboards/utilization-card/area-chart.tsx:131]:
```suggestion
  const removeParenthesesFromLegendData = (legendData) => {
```
Add types

File: packages/shared/src/dashboards/utilization-card/area-chart.tsx [hunk 2]
```diff
@@ -144,7 +153,13 @@ export const AreaChart: React.FC<AreaChartProps> = ({
             themeColor={ChartThemeColor.multiUnordered}
             scale={{ x: 'time', y: 'linear' }}
             padding={{ top: 20, left: 70, bottom: 60, right: 0 }}
-            legendData={chartType ? legendData : null}
+            legendData={
+              chartType
+                ? showLegend
```
Comment (bipuladh) [inline packages/shared/src/dashboards/utilization-card/area-chart.tsx:158]: Where is showLegend coming from and in which cases is it false?

**Review by bipuladh — COMMENTED** [review]
Could you perhaps look into the usage of AreaChart and instead make the changes related to removing parenthesis there? I hardly think making changes to the parenthesis is something that is required in the common packages.

File: packages/shared/src/dashboards/utilization-card/area-chart.tsx (hunk 1 above)
Comment (TimothyAsirJeyasing) [reply packages/shared/src/dashboards/utilization-card/area-chart.tsx:131]: As per UX, we need to show the legend with time, value and the line name (name of the particular line in the graph) in parenthesis. So when we pass the line name with parenthesis as a parameter  to patternfly chart module it also shows the line names in parenthesis at the bottom (bottom line description). So i am passing the name in parenthesis so as to display in the legend and removing them for bottom description.  Also here the variable name legendData is used already so to avoid the compile issue i used a different name. I feel this as a kind of hack only. I do not find any other possibility to solve this issue. Please suggest me if another better possibility to deal this.

File: packages/shared/src/dashboards/utilization-card/area-chart.tsx (hunk 2 above)
Comment (TimothyAsirJeyasing) [reply packages/shared/src/dashboards/utilization-card/area-chart.tsx:158]: Currently the Chart module will display if there are 2 set of data (lines) and will not show any graph if there is only one data series. We have added this showLegend variable to allow showing graph for single line of graph also. Its used only for ceph RBD graph, so i though of using the variable here. One other suggestion i am having is like passing the line name in parenthesis and also to pass the removeParanthesis function to use here. Could you please provide your suggestion ?

File: packages/shared/src/dashboards/utilization-card/area-chart.tsx (hunk 1 above)
Comment (GowthamShanmugam) [reply packages/shared/src/dashboards/utilization-card/area-chart.tsx:131]: Removing removeParenthesesFromLegendData is a specific use case, this whole function is messing up to common logic. For each string operation, you can't keep adding a new function.

File: packages/shared/src/dashboards/utilization-card/area-chart.tsx (hunk 2 above)
Comment (GowthamShanmugam) [reply packages/shared/src/dashboards/utilization-card/area-chart.tsx:158]: i really dont like this logic, just to remove parentheses we are messing up the common logic here

File: packages/shared/src/dashboards/utilization-card/utilization-item.tsx [hunk 3]
```diff
@@ -272,6 +273,7 @@ type UtilizationItemProps = {
   customDateTimeFormatter?: (date: Date) => string;
   description?: string | ((result: PrometheusResult, index: number) => string);
   hideCurrentHumanized?: boolean;
+  hideLink?: boolean;
```
Comment (GowthamShanmugam) [inline packages/shared/src/dashboards/utilization-card/utilization-item.tsx:276]: disableGraphLink

**Comment by SanjalKatiyar** [comment]
update the PR title correctly as per the guidelines...

**Comment by TimothyAsirJeyasing** [comment]
/retitle Remove link and remove cluster name

**Comment by TimothyAsirJeyasing** [comment]
image

File: packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx [hunk 4]
```diff
@@ -253,7 +253,7 @@ export const PVCsSection: React.FC<PVCsSectionProps> = ({
 
 const getDescription = (result: PrometheusResult, _index: number) =>
   // Returning cluster name as a description
-  result.metric?.['cluster'] || '';
+  `${result.metric?.['cluster'] || ''}`;
```
Comment (GowthamShanmugam) [inline packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx:256]: Why this change, it looks no change from old code.

Comment (TimothyAsirJeyasing) [reply packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx:256]: Yes, i will use the earlier code.

**Review by GowthamShanmugam — APPROVED** [review]
